interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -969,6 +971,7 @@ public void exchangeMetadataStreamingCall() throws Exception {
 
   @Test(timeout = 10000)
   public void sendsTimeoutHeader() {
+    Assume.assumeTrue("can not capture server side request headers", server != null);
```

Comment:
out of curiosity, when would server be null?

Reply:
The server is only non-null if the test subclass uses `startStaticServer` to construct a server from the the local process. Setting it up this way lets the developer capture headers and other info on the server side for test assertions. For GAE, the server is already running on a remote host. This sort of test is not applicable and should be skipped.